oe_theme.theme

```diff
@@ -418,6 +420,27 @@ function oe_theme_preprocess_pattern_dialog(array &$variables): void {
   $variables['hidden'] = $variables['hidden'] ? 'true' : 'false';
 }
 
+/**
+ * Implements hook_preprocess_HOOK() for pattern gallery template.
+ */
+function oe_theme_preprocess_pattern_gallery(array &$variables): void {
+  $variables['rows'] = [];
+
+  $count_item = 0;
+  foreach ($variables['items'] as $item) {
+    $item += ['classes' => 'ecl-col-md-3'];
+
+    // Format the icon for ecl.
+    if (!empty($item['icon'])) {
+      $item['icon'] = 'ecl-icon--' . $item['icon'];
+    }
+
+    $item['image'] = ImageMediaValueObject::fromArray($item['image']);
+    $variables['rows'][(int) $count_item / 4][] = GalleryValueObject::fromArray($item);
```

Comment:
Shouldn't we use `ceil($count_item / 4)` here?

Reply:
Resolved on the twig.